docs/For Developer/API/VolatileData.md

```diff
@@ -0,0 +1,69 @@
+# Volatile Data
+
+You can store data in a non-standard manner where it won't be persisted in the event store but will only be persisted in the PostgreSQL database. This type of data is referred to as volatile data, such as controller settings used in user settings. This data is stored only for the user. To store such data, you need a user settings section, a user Id and a JSON object. For example, you could store favorite links from YouTube or the user's preferred topics for Udemy videos. 
+
+## Filtering the Data
+If you are looking for specific user settings, you can filter and sort the results accordingly. The results are returned in a paginated format.
+
+### Usage of the $filter query
+The **$filter** query is used to filter of result-set through restrictions. It can be seen as a where-Clause  in the sense of SQL. The filter can only be applied to properties of the result item. Nested properties are not supported yet. Filter expressions can be combined with an `OR` or an `AND`. In the example below we have a simple data set that represents a user that has only five properties. The filter can be applied of all of these properties.
+
+```json
+{
+    "Name": "Sam, Smith",
+    "FirstName": "Sam",
+    "LastName": "Smith",
+    "CreatedAt": "2022-09-11T09:30:11.6796611+02:00",
+    "Id": 235
+}
+```
+Let say we want all user that have the same last name. In a database there can be user with the same last name. The query would look like:
```

Comment:
> In a database there can be user ...

I don't think the `there` should be _there_ ;)
Suggestion:
> Several user with the same name can be stored in the database.

Reply:
It think you mean user**s**. I  will take the suggestion. It sounds better.